docs/CLIReadme.md

```diff
@@ -2408,6 +2408,127 @@ delimiters are added between records.
 
 * `          $  zowe zos-files upload file-to-uss "file.txt" "/a/ibmuser/my_text.txt"`
 
+<<<<<<< HEAD
+=======
```

Comment:
Is this the result of a merge conflict?
Since Tech Writers can generate this file, maybe we can remove this file from source to avoid confusion.
@BrandonJenkins14, @JamesBauman, What do you think?

Reply:
Sounds OK to me, I don't think we link to it from anywhere.

Reply:
Hi @zFernand0, yes, it was from an outstanding merge conflict. Don't know how it passed. I've fixed it now. Thanks for bringing it up.